bitnami/minio/templates/deployment-standalone.yaml

```diff
@@ -107,6 +107,9 @@ spec:
             {{- if .Values.extraEnv }}
             {{- toYaml .Values.extraEnv | nindent 12 }}
             {{- end }}
+          {{ if .Values.command }}
+          command: {{- toYaml .Values.command |  nindent 10 }} 
```

Comment:
I checked the indent and I think it should be nindent 12. Otherwise you would get yaml formatting issues

Reply:
```suggestion
          command: {{- toYaml .Values.command |  nindent 12 }} 
```